### ISSUE TITLE
localeProjection mistyped in /oas/openapi.yaml

### ISSUE THREAD

**Issue by DJHunn39** [commercetools/commercetools-api-reference#269]
_tl;dr - I can't send multiple localeProjection parameters in a request using the TS SDK because localeProjection is typed as a string instead of an array_

According [to CommerceTools API documentation](https://docs.commercetools.com/api/projects/products-search):
> localeProjection - String - Optional
IETF language tag
.
Enables locale based projection.
**To set more than one locale, use this query parameter multiple times.**

This means that a valid URI including multiple `localeProjections` would include, for example`localeProjection=en-gb&localeProjection=en-us`.

However, the openapi.yaml schema for this field dictates that this field must be a string:

From [the raw file itself](https://raw.githubusercontent.com/commercetools/commercetools-api-reference/main/oas/openapi.yaml):
```yaml
- name: localeProjection
   in: query
   required: false
   style: form
   schema:
       type: "string"
```

I believe this is the cause of an issue in [the TypeScript SDK](https://github.com/commercetools/commercetools-sdk-typescript) where it's not possible to make a GET request including multiple `localeProjection` query parameters without casting around the type to force it to work, i.e I want to make a get request with 2 localeProjections like so:

```ts
.get({
  queryArgs: {
    localeProjection: ['en-gb', 'en-us'],
    ...
  }
});
```

Right now I can't do this as the type of `localeProjection` must be a string.

I believe changing `schema.type` for all `localeProjection` parameter definitions from `string` to `array` will resolve the issue.

**Comment by jenschude** [comment]
Thanks for the report. I opened a PR in the upstream repository with a fix

**Comment by jenschude** [comment]
The change is now applied to the openapi.yml

**Comment by DJHunn39** [comment]
Thanks for the quick reaction to this!

**Comment by jenschude** [comment]
SDKs with fixes have been released

Thanks for the report
